app/src/main/kotlin/no/nav/aap/oppgave/OppgavelisteAPI.kt

```diff
@@ -45,23 +46,16 @@ fun NormalOpenAPIRoute.oppgavelisteApi(dataSource: DataSource, prometheus: Prome
             )
         }
 
+        val enhetService = EnhetService(MsGraphClient(prometheus))
         val token = token()
-        val oppgaverMedTilgang = oppgaver
-            .asSequence()
-            .filter {
-                try {
-                    TilgangGateway.sjekkTilgang(it.tilAvklaringsbehovReferanseDto(), token)
-                } catch (e: Exception) {
-                    log.warn("Feil ved sjekk av tilgang til oppgave ${it.id}", e)
-                    false
-                }
-            }
-            .take(request.maxAntall)
-            .toList()
+        val enhetsGrupper = enhetService.hentEnheter(token.token(), ident())
+        val oppgaverMedTilgang = oppgaver.asSequence().filter {
```

Comment:
Trenger vi `asSequence` på denne? Usikker på om jeg skjønner hva effekten/verdien av å gjøre om lista til sequence er..?

Ser at det var slik før også - så mulig noen andre bør svare på hvorfor dette er en god ide :)

Reply:
Lite googlesøk så får man svar 😂 

> The difference is that when you use a Sequence it will only run the functions if you iterate over the elements.